database/migrations/2019_05_03_000004_add_pause_collection_to_subscriptions_table.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddPauseCollectionToSubscriptionsTable extends Migration
+{
+    /**
+     * Get used table name.
+     *
+     * @return mixed
+     */
+    protected function getTableName()
+    {
+        $modelClass = \Laravel\Cashier\Cashier::$subscriptionModel;
+
+        return (new $modelClass)->getTable();
+    }
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table($this->getTableName(), function (Blueprint $table) {
+            $table->timestamp('resumes_at')->nullable()
+                ->after('quantity');
+            $table->string('pause_behavior', 30)->nullable()
+                ->after('quantity');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down()
+    {
+        Schema::table($this->getTableName(), function (Blueprint $table) {
+            $table->dropColumn('pause_behavior');
+            $table->dropColumn('resumes_at');
+        });
+    }
+}
```

Comment:
Please add the new `resumes_at` column to the existing migration. Existing Cashier users will need to create this column themselves.